cgmanifest.json

```diff
@@ -0,0 +1,54 @@
+{
+  "Registrations": [
+    {
+      "Component": {
+        "Type": "pip",
+        "pip": { "Name": "isort", "Version": "4.2.15" }
+      },
+      "DevelopmentDependency": false
+    },
+    {
+      "Component": {
+        "Type": "pip",
+        "pip": { "Name": "autopep8", "Version": "1.3" }
+      },
+      "DevelopmentDependency": false
+    },
+    {
+      "Component": {
+        "Type": "pip",
+        "pip": { "Name": "pycodestyle", "Version": "2.3.1" }
+      },
+      "DevelopmentDependency": false
+    },
+    {
+      "Component": {
+        "Type": "pip",
+        "pip": { "Name": "pydevd", "Version": "1.9.2" }
```

Comment:
We have to decide how to work if the pydevd version will be kept in sync (in reality the version in pydevd and debugpy would change at each commit as changes are applied in-sync to both repos at the same time and the diff is reviewed prior to inclusion in debugpy -- updating at each commit would be too slow due to the review policies in that case, but I guess that I can create a commit just to increase the pydevd version at the debugpy repo when a release is done).

Reply:
The way we are following now still works. The review process for this has changed a bit. If there are new components "vendored", then we will need to do a review where we may have to go through code scanning (which takes time). Otherwise, we can rely on the automated processing, that should not take time for PRs.